sjb/config/test_cases/test_branch_online_hibernation_unit.yml

```diff
@@ -0,0 +1,11 @@
+---
+parent: 'common/test_cases/origin.yml'
+overrides:
```

Comment:
You need to sync the repo in this job, but otherwise this seems reasonable

Reply:
done, and added a slew of generated files, look good?

Reply:
So here you'd see:

```yaml
overrides:
  sync_repos:
    - name: "online-hibernation"
```

Reply:
Got it, lol, it didn't seem right to add to origin.yml, but that's what I interpreted from your comment. *sigh* 
Do I have it right now? 
Thanks